src/domain/models.py

```
@@ -157,4 +157,110 @@ class FireEvent:
     satellite: str  # MODIS or VIIRS
     brightness_temperature: Decimal
     scan_area: Optional[Decimal] = None  # Area of fire pixel in km²
-    metadata: Dict[str, Any] = field(default_factory=dict)
\ No newline at end of file
+    metadata: Dict[str, Any] = field(default_factory=dict)
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)